abis/ReserveBrickFraxContract.json

```diff
@@ -0,0 +1,345 @@
+[
```

Comment:
is this the same or different from the `FRAX.json` ABI

Reply:
it is different, can remove tho since not being used

Reply:
it is different, can remove tho since not being used